src/utils/sameArrayItems.ts

```diff
@@ -0,0 +1,12 @@
+export const sameArrayItems = <T>(a: T[], b: T[]): boolean => {
```

Comment:
maybe `arraysHaveSameItems`, or something similar. It's easier to read if you have `is` or `can` or `has` in the name, implying the boolean return type.